server/objects/settings/ServerSettings.js

```diff
@@ -79,6 +79,11 @@ class ServerSettings {
     this.authOpenIDGroupClaim = ''
     this.authOpenIDAdvancedPermsClaim = ''
 
+    // Forward Auth
+    this.authForwardAuthPattern = '127.0.0.1/0' // default to exact localhost
```

Comment:
I did not check the Regex, but in any case that must be /32, which will be exactly one address, instead of /0.

Reply:
Ah you're right, I'll fix this tonight.